pages/common/fortune.md

```diff
@@ -0,0 +1,27 @@
+# fortune
+
+> Print a random quotation (fortune-cookie style)
+
+- Print a quotation
+
+`fortune`
+
+- Print a quotation from a given database
+
+`fortune {{database}}`
+
+- Print a list of quotation databases
```

Comment:
Hmm, interesting. So there's a plaintext version of the quotations list and a binary one, I assume for quick lookup? Is that the case in general? It could be just in your implementation (I'm not on a Linux box, so I can't test that hypothesis myself).

Nevertheless, both are still files and not proper databases, technically speaking, so perhaps "dictionary" could work; but honestly I can't imagine that one would assume such a setup just from the word "dictionary", in which case we'd be only inducing confusion. So in the end, it may be better to just say the bare minimum and technical truth (they are indeed files in the filesystem). Which has the added bonus of matching the option name :)